wire-ios/Wire-iOS/Sources/UserInterface/UserProfile/Devices/UserClientCell.swift

```diff
@@ -23,142 +23,174 @@ import WireDataModel
 
 final class UserClientCell: SeparatorCollectionViewCell {
 
-    typealias IconColors = SemanticColors.Icon
-    typealias ViewColors = SemanticColors.View
-    typealias LabelColors = SemanticColors.Label
+    enum EdgeInsetConstants {
+        static let small: CGFloat = 2.0
+        static let medium: CGFloat = 4.0
+        static let normal: CGFloat = 8.0
+        static let `default`: CGFloat = 16.0
+    }
 
-    private let titleLabel = DynamicFontLabel(fontSpec: .bodyTwoSemibold,
-                                              color: LabelColors.textDefault)
-    private let subtitleLabel = DynamicFontLabel(fontSpec: .mediumRegularFont,
-                                                 color: LabelColors.textCellSubtitle)
+    typealias LabelColors = SemanticColors.Label
+    typealias IconColors = SemanticColors.Icon
 
-    private let deviceTypeIconView = UIImageView()
+    // MARK: - Properties
+    let nameLabel = DynamicFontLabel(style: .headline,
+                                     color: LabelColors.textDefault)
+    let labelLabel = DynamicFontLabel(style: .subheadline,
```

Comment:
suggestion: `labelLabel` isn't descriptive enough. I would suggest a name like: `userClientLabel`

Reply:
done

Reply:
don't need it anymore